### DISCUSSION TITLE
suggestion: larger menu text and larger tabs

### DISCUSSION THREAD

**Discussion by mattlorig** [sumatrapdfreader/sumatrapdf#3852] [Ideas]
Just as the title suggests ... I think it would be helpful to have larger text in the menu and larger tabs.  I find the current size of text to be somewhat difficult to read.

**Comment by Back-T0** [comment]
When it comes to tab size, the cross on the tab every time I close it is not easy to click because it is too small (although the shortcut key can be easily done), I wonder if Sumatra will be able to double-click the tab to close the tab like Edge browser in the future.

**Reply by GitHubRulesOK** [reply]
@Back-T0 
SumatraPDF should respect middle click tab (as long as there is enough tab without the close button :-)
the close button should be highly visible when cursor is over it
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5607021d-4999-4b08-b4ee-a76483bd553e)

**Reply by Back-T0** [reply]
I just realized that I can middle-click to close! Thank you.
